jemalloc-sys/src/pthread_atfork.c

```diff
@@ -0,0 +1,13 @@
+#include <pthread.h>
```

Comment:
I don't think you need to include this header.

Reply:
Removed.